node/core/pvf-checker/src/lib.rs

```diff
@@ -536,9 +537,30 @@ async fn initiate_precheck(
 ) {
 	gum::debug!(target: LOG_TARGET, ?validation_code_hash, ?relay_parent, "initiating a precheck",);
 
+	let (ee_params_tx, ee_params_rx) = oneshot::channel();
+	sender
+		.send_message(RuntimeApiMessage::Request(
+			relay_parent,
+			RuntimeApiRequest::SessionEeParamsByParentHash(ee_params_tx),
+		))
+		.await;
+
+	// FIXME: Error checking?
+	let ee_params = match ee_params_rx.await {
+		Err(_) => return,
+		Ok(Err(_)) => return,
+		Ok(Ok(Some(eep))) => eep,
+		Ok(Ok(None)) => return,
+	};
```

Comment:
@pepyakin one specific point: my mind has somewhat blown when I was trying to decide what to do in case of failure at this place. Basically we have two kinds of failure here: runtime API failure to return a result (which likely means we're running an old runtime not yet supporting EE parameters) and data absence failure (which means things are going really wrong). I'm not sure how to manage PVF checker state in each of those cases.